docs/release-process.md

```diff
@@ -175,21 +165,21 @@ back-port some fixes (e.g. a security fix) and (backwards compatible) changes
 from an upcoming release and release them without also releasing all the other
 (potentially breaking) changes.
 
-To make the following steps easier, set the `RELEASE_BRANCH` environment
-variable to the name of the stable branch of the `YY.MINOR` release you want
-to back-port the changes to, e.g. `stable/20.11.x`:
+Set the `RELEASE_BRANCH` environment variable to the name of the stable branch
+of the `YY.MINOR` release you want to back-port the changes to, e.g.
+`stable/21.2.x`, and export it:
 
 ```bash
-RELEASE_BRANCH="stable/20.11.x"
+export RELEASE_BRANCH="stable/21.2.x"
 ```
 
 ### Back-port Changes
 
-Create a new branch, e.g. `<GITHUB-NAME>/${RELEASE_BRANCH}/backport-foo`, from
```

Comment:
Hm I think we should still namespce the changes by user, similar as we do for regular branches.

Reply:
Yeah, this is a fair point. I was balancing the need for namespacing and having copy & paste Release Process docs.

My internal reasoning for not bothering with namespacing is that these branches (backport and changelog branches) should typically be short-lived and deleted automatically.